java-10/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:10-jdk-slim
+
+RUN apt-get update && apt-get install -y wget
+RUN wget -O /dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.1/dumb-init_1.2.1_amd64
+RUN chmod +x /dumb-init
+
+ENV LC_ALL="no_NB.UTF-8"
+ENV LANG="no_NB.UTF-8"
+ENV TZ="Europe/Oslo"
+
+# Please see https://blogs.oracle.com/java-platform-group/java-se-support-for-docker-cpu-and-memory-limits
+ENV DEFAULT_JAVA_OPTS="-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
```

Comment:
Disse JVM flaggene kan fjernes for JDK10 da de er deprecated. Se https://bugs.openjdk.java.net/browse/JDK-8146115 og https://bugs.openjdk.java.net/browse/JDK-8194086